packages/react-next/src/components/ContextualMenu/examples/ContextualMenu.CustomizationWithNoWrap.Example.tsx

```diff
@@ -26,7 +26,7 @@ export const ContextualMenuCustomizationWithNoWrapExample: React.FunctionCompone
             checkForNoWrap: true,
           },
           items: [
-            { key: 'bulb', text: 'Lightbulb', onRender: renderCharmMenuItem, className: classNames.item },
+            { key: 'bulb', text: 'LightBulb', onRender: renderCharmMenuItem, className: classNames.item },
```

Comment:
This was correct (with respect to the actual icon name) as-is.
```suggestion
            { key: 'bulb', text: 'Lightbulb', onRender: renderCharmMenuItem, className: classNames.item },
```